utils/templates/query.py.tpl

```diff
@@ -0,0 +1,352 @@
+#  Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This template is somewhat complex, because it has a farly significant manually generated code at the top, which are the generic parts of the file up to and including the `Query` base class.